### DISCUSSION TITLE
Cat boost and KS statistics

### DISCUSSION THREAD

**Discussion by santoshenrique2021** [pycaret/pycaret#3685] [Q&A] [answered]
Considering classification, I would like to know how to add catboost in pycaret. Furthermore, could you tell me how to compute KS statistics in pycaret?

Best regards

**Comment by celestinoxp** [comment, marked as answer]
pycaret automatically detects if you have catboost installed. Then you just need to install `pip install catboost`

You can use `create_model('catboost')` or `compare_models()` to check several models and compare metrics

You can check doc: https://pycaret.readthedocs.io/en/stable/api/classification.html

**Reply by santoshenrique2021** [reply]
Thank you.
I got success.
